View/ViewHandler.php

```diff
@@ -533,4 +546,19 @@ private function getDataFromView(View $view)
             )
         );
     }
+
+    /**
+     * Triggers a deprecation if a getter is extended.
+     *
+     * @todo remove this in 2.0.
+     */
+    private function deprecateGetter($name)
+    {
+        if (is_subclass_of($this, __CLASS__)) {
+            $method = new \ReflectionMethod($this, $name);
+            if ($method->getDeclaringClass()->getName() != __CLASS__) {
+                @trigger_error(sprintf('%s::%s() is deprecated since version 1.8 and will be removed in 2.0.', __CLASS__, $name), E_USER_DEPRECATED);
```

Comment:
You should mention that overwriting it is deprecated (and maybe mention ``get_class($this)`` in the message too to help finding where it happens)